### PR TITLE
Update glorot_uniform_initializer to match other Initializers

### LINKED ISSUE #19910: glorot_uniform_initializer and glorot_normal_initializer not consistent with the rest of the initializers?
In /ops/init_ops.py:

All the initializers inherit from class Initializer and implements:
def __call__(self, shape, dtype=None, partition_info=None)

This gives a different signature from the two glorot initializer functions (which are defs rather than classes), and the parameter 'partition_info' doesn't exist. This causes compatibility problems when passing them in as initializers.

### PR DESCRIPTION
**Pull request by yongtang**
This fix tries to address the issue raised in #19910 where the signature of glorot_uniform_initializer does not match other Initializers (e.g., random_uniform_initializer).

This fix update the glorot_uniform_initializer so that it matches the signature in a similiar way as other
Initializers.

This fix fixes #19910.

Signed-off-by: Yong Tang <yong.tang.github@outlook.com>

### CODE REVIEW

**Comment by qlzh727** [comment]
Nudge @fchollet for review/approval.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @fchollet: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 29 days with no activity and the `awaiting review` label has been applied.

**Comment by qlzh727** [comment]
Sorry for the late reply. Please resolve the merge conflict, and fix the api compatibility test failure. See  https://source.cloud.google.com/results/invocations/4de5073a-8d47-4424-a7fa-bef0a215d19c/targets/%2F%2Ftensorflow%2Ftools%2Fapi%2Ftests:api_compatibility_test/log

**Review by fchollet — APPROVED** [review]
Thanks for the PR! Changes look good to me.

@qlzh727 we'll need an API owner to validate the API changes in this PR.

**Comment by qlzh727** [comment]
Ack, I will probably patch the PR internally and update the pbtxt.

**Comment by qlzh727** [comment]
Sure. Will cc @martinwicke on the change.

File: tensorflow/python/ops/init_ops.py [hunk 1]
```diff
@@ -1178,16 +1168,45 @@ def glorot_normal_initializer(seed=None, dtype=dtypes.float32):
...
+uniform_unit_scaling_initializer = UniformUnitScaling
+variance_scaling_initializer = VarianceScaling
+glorot_uniform_initializer = GlorotUniform
+glorot_normal_initializer = GlorotNormal
+orthogonal_initializer = Orthogonal
+identity_initializer = Identity
+convolutional_delta_orthogonal = ConvolutionDeltaOrthogonal
+convolutional_orthogonal_1d = ConvolutionOrthogonal1D
+convolutional_orthogonal_2d = ConvolutionOrthogonal2D
+convolutional_orthogonal_3d = ConvolutionOrthogonal3D
```
Comment (martinwicke) [inline tensorflow/python/ops/init_ops.py]: While we're here, could we remove this aliases (are they even effective? Do they do anything given that the tf_export decorators determine what's in the API?)?

@annarev do you think these actually do anything?

**Comment by martinwicke** [comment]
@tensorflow/api-owners This is an almost-noop in terms of API, I think it's fine.

File: tensorflow/python/ops/init_ops.py (hunk 1 above)
Comment (annarev) [reply tensorflow/python/ops/init_ops.py]: Looks like these are used in contrib:
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/contrib/framework/__init__.py
I suppose they can be imported with "as convolutional_orthogonal_*d" instead.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @qlzh727: It has been 16 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Pull request merged**